langchain4j-core/src/main/java/dev/langchain4j/internal/InvocationPolicy.java

```diff
@@ -0,0 +1,47 @@
+package dev.langchain4j.internal;
+
+import java.util.concurrent.Callable;
+
+import static dev.langchain4j.internal.RetryUtils.DEFAULT_RETRY_POLICY;
+
+public interface InvocationPolicy {
```

Comment:
Should probably mark this with `@FunctionalInterface`?

Reply:
Done, thanks.